packages/dx-grid-core/src/utils/merge-sort.ts

```diff
@@ -57,8 +57,8 @@ export default (
     return 0;
   },
 ) => {
-  const result = array.slice();
-  const auxiliary = array.slice();
+  const result = Array.prototype.slice.call(array);
```

Comment:
From the code of `merge` it seems that it does not modify `result`, isn't it?

Reply:
Tests display other, or I don't understand something?

Reply:
My mistake - the exported function itself modifies the `array`. So copy is required.